Main.java

```diff
@@ -0,0 +1,27 @@
+public class Main{
```

Comment:
Only submit formatted source code!
VSCode: https://stackoverflow.com/questions/29973357/how-do-you-format-code-in-visual-studio-code-vscode
IntelliJ: https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html